lib/utp.js

```diff
@@ -0,0 +1,13 @@
+module.exports = (() => {
+  try {
+    const utp = require('utp-native')
+    utp.UTP_SUPPORT = true
```

Comment:
This modifies the exported value of the package, even for users who `require('utp-native')` in their own app since Node.js caches the exported object. Ideally, we shouldn't modify it.

Can you eliminate the  `UTP_SUPPORT ` property from this file and just export either the result of `require('utp-native')` or an empty object? This way, we treat both cases -- (1) missing uTP support in Node.js and (2) running in the browser -- in the same way. In either of these cases, running `require('./lib/utp')` will return an empty object.

You can still export UTP_SUPPORT if you want, but I'd do it on the WebTorrent object the way we do for `WebTorrent.WEBRTC_SUPPORT`.